examples/warm_up_example/warmed_up_fenda/README.md

```diff
@@ -0,0 +1,34 @@
+# Warmed Up FENDA Federated Learning Example
+This example provides an example of training a FENDA type model on a non-IID subset of the MNIST data after being warmed up with FedAvg. The FL server expects three clients to be spun up (i.e. it will wait until three clients report in before starting training). Each client has a modified version of the MNIST dataset. This modificThis modification essentially subsamples a certain number from the original training and validation sets of MNIST in order to synthetically induce local variations in the statistical properties of the clients training/validation data. In theory, the models should be able to perform well on their local data while learning from other clients data that has different statistical properties. The proportion of labels at each client is determined by dirichlet distribtuion across the classes. The lower the beta parameter is for each class, the higher the degree of the label heterogeneity.
+
+The server has some custom metrics aggregation and uses Federated Averaging as its server-side optimization. The implementation uses a special type of weight exchange based on named-layer identification.
+
+After the warm-up training, clients can load their warmed-up models and continue training with the FENDA algorithm. To maintain consistency in the data loader between both runs, it is crucial to set a fixed seed for both clients and the server, ensuring uniformity in random data points across consecutive runs. Therefore, we ensure a fixed seed is set for these consecutive runs in both the `client.py` and `server.py` files. Additionally, to load the warmed-up models, it's important provide the path to the pretrained models and unique number of each client. In this particular scenario, we assigned a unique number between 1 and 3 to each client, ensuring that we can load the trained local model for each client from the previous example as a warmed-up model. Since models in the two runs can be different, loading weights from the pretrained model requires providing a mapping between the pretrained model and the model used in FL training. This mapping is accomplished through the `weights_mapping.json` file, which contains the names of the pretrained model's layers and the corresponding names of the layers in the model used in FL training.
```

Comment:
Perhaps we can also describe what we're doing specifically in this example. That is, we're taking the components of a single model trained with FedAvg and loading the weights into both the local and global submodules of a FENDA model.